.github/workflows/tests.yml

```
@@ -21,7 +21,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['8.0']
+        php-versions: ['8.1']
```

We must support both so users can migrate from one PHP version to the next one without big of a down time.
```suggestion
        php-versions: ['8.0', '8.1']
```

---

I thought we wanted to upgrade the minimum version to php 8.1 because the internal ticket mentions updating the version in composer.json as well?

I agree we should support both to make transitioning easier. 8.0 is still officially supported by php.

---

Well, the ticket https://mautic.atlassian.net/browse/TPROD-457 is called "Mautic runs on PHP 8.0 and 8.1" :)